Shipr-Helm/templates/shipr-payment-deployment.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ Values.payment.name }}
+  labels:
+    app: {{ Values.payment.name }}
+spec:
+  replicas: {{ Values.payment.replicas }}
+  selector:
+    matchLabels:
+      app: {{ Values.payment.name }}
+  template:
+    metadata:
+      labels:
+        {{- with .Values.payment.labels }}
+        {{- toYaml . | nindent 6 }}
+        {{- end }}
+    spec:
+      containers:
+      - name: {{ Values.payment.name }}
+        image: {{ Values.payment.repository }}:{{ Values.payment.tag }}
+        ports:
+        - containerPort: {{ Values.payment.containerPort }}
+        env:
+          - name: {{ Values.payment.env.host-name }}
+            value: {{ Values.payment.env.host-value }}
+          - name: {{ Values.payment.env.port-name }}
+            value: {{ Values.payment.env.port-value }}
+          - name: {{ Values.payment.env.db-name }}
+            value: {{ Values.payment.env.db-value }}
+          - name: {{ Values.payment.env.url-name }}
+            value: {{ Values.payment.env.url-value }}
+          - name: {{ Values.payment.env.fee-name }}
+            value: {{ Values.payment.env.fee-value }}
```

Comment:
I think this approach is not scalable, what If the dev team adds 10 more variables in next sprint?

Reply:
Used configmap for the variables